### PR TITLE
JS: Add taint steps for markdown parsers

### PR DESCRIPTION
**Pull request by erik-krogh**
Adds taint-steps for markdown to HTML converters that doesn't sanitize their outputs. 

[Evaluation looks fine](https://github.com/dsp-testing/erik-krogh-BCQS/tree/auto/data/workflow/mark-eval-1/reports), with one new TP.

### CODE REVIEW

File: javascript/change-notes/2021-02-10-markdown.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+lgtm,codescanning
+* The dataflow libraries now model dataflow in markdown parses.
```
Comment (intrigus-lgtm) [inline javascript/change-notes/2021-02-10-markdown.md:2]:
```suggestion
* The dataflow libraries now model dataflow in markdown parsers.
```

**Comment by max-schaefer** [comment]
OOI, I see you are using local data flow instead of API graphs. Is that due to performance reasons or ergonomics?

**Comment by erik-krogh** [comment]
> OOI, I see you are using local data flow instead of API graphs. Is that due to performance reasons or ergonomics?

Ergonomics.  
I would have to create a bunch of `API::EntryPoint` to support references to global variables.

File: javascript/change-notes/2021-02-10-markdown.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+lgtm,codescanning
+* The dataflow libraries now model dataflow in markdown parsers.
```
Comment (asgerf) [inline javascript/change-notes/2021-02-10-markdown.md:2]:
```suggestion
* The security queries now track taint through markdown parsers.
```

**Review by asgerf — COMMENTED** [review]
Sorry, this fell under my radar for a bit. LGTM, just a minor comment in the change note.

**Review by asgerf — APPROVED** [review]

**Pull request merged**